examples/docker-compose.yml

```diff
@@ -1,16 +1,14 @@
 version: '2'
 services:
   conjur:
-    image: registry.tld/conjur-appliance-cuke-master:4.9-stable
+    image: registry2.itci.conjur.net/conjur-appliance-cuke-master:4.9-stable
```

Comment:
Right; I first worked from the assumption that wrong endpoint was being used. Thanks for fixing that! (BTW, I really wish we'd rename `registry.tld`.)